libs/config.js

```diff
@@ -42,22 +42,42 @@ const SAMPLE = `
       "adminURL": "http://localhost:6060",
       "adminAuth": ""
     }
-  }
+  },
+  "domainToEnv": {}
 }
 `
 
 const fs = require('fs')
 
 let config
 
+const domainToEnv = {
+  'cozy.wtf': 'dev',
+  'cozy.blue': 'dev',
+  'cozy.red': 'int',
+  'cozy.works': 'int',
+  'cozy.company': 'int',
+  'cozy.rocks': 'stg',
+  'mycozy.cloud': 'prod',
+  'cozyorange.cloud': 'prod',
+  'mytoutatice.cloud': 'prod',
+  'tools:8080': 'local'
```

Comment:
This is an opportunity to add : 'localhost:8080':'local' since we're now using cozy.localhost instead of cozy.tools :)